### DISCUSSION TITLE
Raw/baseline values for tomcat and java.lang

### DISCUSSION THREAD

**Discussion by abhyp** [glowroot/glowroot#1234] [Q&A] [answered]
How to check what raw/baseline values for tomcat,java.lang set in Glowroot central

**Comment by Never-lab** [comment, marked as answer]
## Triage

This is a **configuration how-to**, not a feature request.

"Raw" vs delta values = the **Counter?** flag on gauge MBean attributes (`counter: true` in `config.json`). There is no separate "baseline" setting in Glowroot.

- **java.lang** gauges: from built-in JVM gauge config / `uiDefaults.defaultGaugeNames`
- **tomcat** gauges: not preconfigured — add Catalina/Tomcat MBeans manually under Config → Gauges

Answer moved to Discussions → Q&A: https://github.com/glowroot/glowroot/discussions/1228

**Verdict:** close as Q&A (wrong channel). Re-label: `discussion` → redirect to Discussions.

---
*Community triage*
